pages/polling/review.tsx

```diff
@@ -73,20 +77,34 @@ const PollingReview = ({ polls }: { polls: Poll[] }) => {
                 {bpi <= 2 && <SubmitButton />}
                 {bpi <= 2 && !!account && <ReviewBox polls={polls} activePolls={activePolls} />}
                 <Stack sx={{ display: activePolls.length ? undefined : 'none' }}>
-                  {Object.keys(ballot).map((pollId, index) => {
-                    const poll = findPollById(polls, pollId);
-
-                    if (!poll) {
-                      return null;
-                    }
+                  {votedPolls.map((poll, index) => {
                     return (
                       <PollOverviewCard
                         key={poll.multiHash}
                         poll={poll}
                         reviewPage={true}
                         showVoting={true}
                         sx={cardStyles(index, ballotLength)}
-                      />
+                      >
+                        <Box sx={{ pt: 2 }}>
+                          <Text
+                            as="p"
+                            variant="h2"
+                            sx={{
+                              fontWeight: 'semiBold',
+                              mb: 2
+                            }}
+                          >
+                            Why are you voting for this proposal?
+                          </Text>
+                          <Textarea
+                            value={comments.find(i => i.pollId === poll.pollId)?.comment}
+                            onChange={e => updateComment(e.target.value, poll.pollId)}
+                            placeholder="Optional. 250 character max. You'll be prompted to sign a message with your wallet."
```

Comment:
We should match the comment inputs between exec and polls. We recently got rid of the placeholder and added the text below.
<img width="775" alt="Screen Shot 2021-12-15 at 11 03 20 AM" src="https://user-images.githubusercontent.com/5225766/146166051-7e0b3d23-0189-48bb-b694-28cb0fc3c3ce.png">